### PR TITLE
[1LP][RFR] manual timelines tests

### PR DESCRIPTION
**Pull request by izapolsk**
1. Added manual timelines tests.
There should be much more tests but the rest will be added later.
2. Fixed manual py test plugin to correctly collect/uncollect manual test cases.
The collection works in the following way now:
If --manual argument is added then pytest collects only manual test cases.
If --manual argument isn't added then pytest ignores all manual test cases.

### CODE REVIEW

**Comment by izapolsk** [comment]
Please ignore PRT errors those aren't related to these changes and caused by problems with providers.

File: markers/manual.py [hunk 1]
```diff
@@ -18,10 +18,13 @@ def pytest_collection_modifyitems(session, config, items):
...
-        if config.getvalue('manual') and not item.get_marker('manual'):
-            continue
+        if config.getvalue('manual'):
+            if not item.get_marker('manual'):
+                continue
         else:
-            new_items.append(item)
+            if item.get_marker('manual'):
+                continue
+        new_items.append(item)
```
Comment (mfalesni) [inline markers/manual.py]: I was thinking ...

```python
for item in items:
    if bool(config.getvalue('manual')) == bool(item.get_marker('manual')):
        new_items.append(item)
```

Comment (RonnyPfannschmidt) [inline markers/manual.py]: **REQUIRED**:

* call to the `pytest_deselected` hook
* partition by the marker value, select the partition by the config option

Comment (mfalesni) [inline markers/manual.py]: Or even as a one liner :)
```python
manual = bool(config.getvalue('manual'))
items[:] = [item for item in items if manual == bool(item.get_marker('manual'))]
```

Comment (izapolsk) [inline markers/manual.py]: @RonnyPfannschmidt , @psav , @mfalesni, I optimized this code according to you request.
It's a pity that python doesn't have eqv operator. So, I had to replace it with "not xor".

**Comment by dajoRH** [comment]
Lint report for commit c94e70eebcb06835bb370ca746212422bb34367e:

`markers/manual.py`:
- :red_circle: Line 4:1: **F401** *'operator.xor' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**